WORKSPACE

```diff
@@ -11,9 +11,9 @@ http_archive(
 # ===== gtest =====
 new_http_archive(
     name = "com_google_googletest",
-    url = "https://github.com/google/googletest/archive/release-1.8.0.zip",
-    sha256 = "f3ed3b58511efd272eb074a3a6d6fb79d7c2e6a0e374323d1e6bcbcc1ef141bf",
-    strip_prefix = "googletest-release-1.8.0",
+    url = "https://github.com/google/googletest/archive/master.zip",
```

Comment:
It should say something like 'https://github.com/google/googletest/archive/88f0493098c8d9fd0f096c2158a0e56deb952d53.zip'

'master' is just the latest.

Reply:
OK, Thanks a lot! I will submit a new.

But by what command can I know that the latest master zip is 88f0493098c8d9fd0f096c2158a0e56deb952d53.zip? hope to get your guidance. 

Thanks
Bo

Reply:
Use the command-line tool `sha256`.